main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"github.com/Gauravsinghal09/Ecommerce/Config"
+	"github.com/Gauravsinghal09/Ecommerce/Controllers"
+	"github.com/Gauravsinghal09/Ecommerce/Models/Customer"
+	"github.com/Gauravsinghal09/Ecommerce/Models/Order"
+	"github.com/Gauravsinghal09/Ecommerce/Models/Product"
+	"github.com/Gauravsinghal09/Ecommerce/Router"
+	"gorm.io/driver/mysql"
+	"gorm.io/gorm"
+	"gorm.io/gorm/logger"
+)
+
+func InitialiseDB() (*gorm.DB, error) {
```

Comment:
a piece of code should be where you expect it to find
. should move it to database.php